pyccel/ast/basic.py

```diff
@@ -6,14 +6,14 @@
 """
 This module contains classes from which all pyccel nodes inherit.
 They are:
-- Basic, which provides a python AST
-- PyccelAstNode which describes each PyccelAstNode
+- PyccelAstNode, which provides a python AST
+- TypedAstNode which describes each TypedAstNode
```

Comment:
Not sure what this line was supposed to read...

Reply:
Hmmm, this is definitely unreadable. I suggest:
``` 
- PyccelAstNode, which provides a base class for our Python AST nodes
 - TypedAstNode which provides a base class for AST nodes which require type descriptors 
```

Reply:
Almost perfect. Isn't `TypedAstNode` a subclass of `PyccelAstNode`?

Reply:
Yes
```
- PyccelAstNode, which provides a base class for our Python AST nodes
- TypedAstNode which inherits from PyccelAstNode and provides a base class for AST nodes which require type descriptors 
```

Reply:
Please apply the change!

Reply:
Done in 1c96320.